tools/gcp/comp_test/run_comp_test.py

```diff
@@ -0,0 +1,147 @@
+#!/usr/bin/env python2.7
```

Comment:
IMHO, this is just way too complicated for what it's doing.

After running, run_interop_tests.py --manual_run, you basically have a list of dockerimages you could just push to GCR, but instead you use a 100line script to "enhance" the existing docker images with 3 extra files and only then you push.

To run the tests, you pull the docker images from GCR and the first thing you do is you you extract those files from the docker images using another 100line script (and you need to patch the client commands file because you changed the tags of the docker images before).

Besides doing unnecessary work, I feel this still doesn't solve the question of managing which docker images/grpc version that will get tested and where will be the list of versions to be tested stored.

Example: 
with these scripts, you would run e.g. `run_comp_test.py --gcr_tag java_cloud_to_prod:v1.0.0`, but that's pretty much equivalent as keeping the interop_clients_cmds.sh  and renaming it to `interop_clients_cmds_java_cloud_to_prod_v1.0.0.sh' - excepts for your approach you need to introduce 3 new python scripts that someone has to understand and maintain.